.github/workflows/run-robot-tests.yaml

```diff
@@ -56,9 +56,9 @@ jobs:
         # Run Robot Framework tests in REST mode from test/robot directory
         run: robot test/robot
       # Run Robot Framework tests in Python mode against running docker compose
-      - name: Run Robot Framework tests (Python mode)
-        # Run Robot Framework tests in Python mode from test/robot directory
-        run: TEST_MODE=Python robot test/robot/MRandLogicalModel.robot
+      # - name: Run Robot Framework tests (Python mode)
+      #   # Run Robot Framework tests in Python mode from test/robot directory
+      #   run: TEST_MODE=Python robot test/robot/MRandLogicalModel.robot
```

Comment:
Any reason for leaving this commented instead of just erasing them?

Reply:
no reason actually, we can drop that too.